### ISSUE TITLE
Command line interfaces

### ISSUE THREAD

**Issue by jhorman** [cxxr/better-java#8]
Wanted to suggest Airline, for a command line interface builder. Python developers are used to using something like clize for this.

https://github.com/airlift/airline

**Comment by cxxr** [comment]
Yeah I think mentioning that and [Commons CLI](http://commons.apache.org/proper/commons-cli/) would be a good addition.

**Comment by daveloyall** [comment]
I haven't tried those two, but I've used http://www.jcommander.org/.  ...It's a pain in the butt, but it provides an end-user experience that closely matches the standard unix way of handling command line options (-s == --short, etc).
